File: benchmark/results/java-invocation-script/run-004/a-mcp/output/billing/value_objects/TaxLine.java
package com.metaengine.demo.billing.value_objects;

import java.time.OffsetDateTime;

// TaxLine value object.

public class TaxLine {
    public String id;
    public OffsetDateTime createdAt;
    public OffsetDateTime updatedAt;
    public String name;
    public String description;

    public TaxLine(String id, OffsetDateTime createdAt, OffsetDateTime updatedAt, String name, String description) {
        this.id = id;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
        this.name = name;
        this.description = description;
    }
}

File: benchmark/results/python-multilang/run-002/b-baseline/output/metaengine_demo/billing/value_objects/tax_line.py
"""TaxLine value object for the billing domain."""

from dataclasses import dataclass
from datetime import datetime


@dataclass(frozen=True)
class TaxLine:
    """Immutable line item describing applied tax."""

    id: str
    createdAt: datetime
    updatedAt: datetime
    name: str
    description: str
